iodata/formats/xyz.py

```diff
@@ -91,15 +91,125 @@
 """
 
 
+BOOL_MAP = {"T": True, "F": False, "True": True, "False": False}
+
+
+def convert_title_value(value: str):
+    """Search for the correct dtype and convert the string."""
+    list_of_splits = value.split()
+    # If it is just one item, first try int, then float and finally return a bool or str
+    if len(list_of_splits) == 1:
+        try:
+            converted_value = int(value)
+        except ValueError:
+            try:
+                converted_value = float(value)
+            except ValueError:
+                if value in BOOL_MAP.keys():
+                    converted_value = BOOL_MAP[value]
+                else:
+                    converted_value = value
+    else:
+        # Do the same but return it as a numpy array
+        try:
+            converted_value = np.array(list_of_splits, dtype=np.int)
+        except ValueError:
+            try:
+                converted_value = np.array(list_of_splits, dtype=np.float)
+            except ValueError:
+                try:
+                    converted_value = np.array([BOOL_MAP[bool_key] for bool_key in list_of_splits],
+                                               dtype=np.bool)
+                except KeyError:
+                    converted_value = np.array(list_of_splits, dtype=np.str)
+    return converted_value
+
+
+def parse_properties(properties: str):
+    """Parse the properties into atom_columns."""
+    atom_columns = []
+    # Maps the dtype to the atom_columns dtype, load_word and dump_word
+    dtype_map = {"S": (np.dtype('U25'), str, "{:10s}".format),
+                 "R": (float, float, "{:15.10f}".format),
+                 "I": (int, int, "{:10d}".format),
+                 "L": (bool, lambda word: BOOL_MAP[word], lambda boolean: "T" if boolean else "F")}
+    # Some predefined iodata attributes which can be mapped
+    # Only pos is assumed to be in angstrom, no unit convertion takes place for the other attributes
+    atom_column_map = {'pos': ('atcoords', None, (3,), float,
+                               (lambda word: float(word) * angstrom),
+                               (lambda value: "{:15.10f}".format(value / angstrom))),
+                       'mass': ('atmasses', None, (), float, float, "{:10.5f}".format),
```

Comment:
ASE uses atomic mass units by default. Hence, convertion to atomic units of mass should take place in IOData. I have added a column of masses in `water_extended_trajectory.xyz` as created by ASE. Furthermore, ASE seems to use the property 'masses' for the atomic masses instead of the documented 'mass'. All those things are added in the last commit.